python/ray/serve/scripts.py

```diff
@@ -373,6 +373,12 @@ def shutdown(address: str, yes: bool):
     ),
 )
 def build(import_path: str, app_dir: str, output_path: Optional[str]):
+
+    if output_path is not None and not output_path.endswith(".yaml"):
+        raise ValueError(
+            f'FILE_PATH must end with ".yaml". Got "{output_path}" instead.'
+        )
```

Comment:
this is too strict. we should not force a suffix.

Reply:
Sounds good, I removed the check.